cdap-validation/bin/input.json

```diff
@@ -0,0 +1,10 @@
+{
+  "params":
+  {
+    "verbose": "default",
+    "uri": "http://ambari.prod.continuuity.net:8080/",
+    "userpass": "admin:admin",
```

Comment:
Umm... Operational security is out the window. I am going to go break this script (change the password on Ambari), now.